source/_integrations/androidtv_remote.markdown

```diff
@@ -208,7 +134,7 @@ target:
   entity_id: remote.living_room_tv
 ```
 
-### Dashboard example
+## Dashboard example
```

Comment:
Same here. I think ### would be better.

Reply:
Same as https://github.com/home-assistant/home-assistant.io/pull/27308#discussion_r1188366143
I just reverted the changes before https://github.com/home-assistant/home-assistant.io/pull/27059.